spec/support/seeds/e2e/initial_e2e_db.rb

```diff
@@ -0,0 +1,106 @@
+return if Rails.env.production?
```

Comment:
I don't need the same seed data as we have for our development environment. For now I've added an initial admin user and another user to do some user tests. If we need more seed data, we can pull some stuff from the development environment seed data and copy it in here.